app/controllers/v2/tailorings_controller.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module V2
+  # API for Tailorings
+  class TailoringsController < ApplicationController
+    def index
+      render_json tailorings
+    end
+    permission_for_action :index, Rbac::POLICY_READ
+
+    def show
+      render_json tailoring
+    end
+    permission_for_action :show, Rbac::POLICY_READ
+
+    def tailoring_file
+      send_data(
+        xccdf_tailoring_file,
+        filename: xccdf_tailoring_filename,
+        type: Mime[:xml]
+      )
+    end
+    permission_for_action :tailoring_file, Rbac::POLICY_READ
+    permitted_params_for_action :tailoring_file, id: ID_TYPE.required
+
+    private
+
+    def tailorings
+      @tailorings ||= authorize(resolve_collection)
+    end
+
+    def tailoring
+      @tailoring ||= authorize(expand_resource.find(permitted_params[:id]))
```

Comment:
Changes from https://github.com/RedHatInsights/compliance-backend/pull/1931 seem to make a problem here.

Reply:
Nope, you did a bad rebase